### ISSUE TITLE
Permissions on created directories are sometimes wrong

### ISSUE THREAD

**Issue by jan-hofmeier** [StroopwafelCFW/wafel_setup_mlc#4]

**Comment by jan-hofmeier** [comment]
On a good console:
```python
>>> [ hex(p) for p in w.FSA_GetInfoByQuery(w.get_fsa_handle(), '/vol/storage_mlc01/usr/save/system/nim', 5)[1]]
['0xfffcffe9', '0x8', '0x0', '0x1d27af80', '0x8', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x10', '0x0', '0x0', '0x0', '0xbabe0000', '0x90', '0x1d27b040', '0x1d27b9a0', '0xbabe0002', '0x40', '0x1d27b0a0', '0x0', '0x1d27b0f0']
>>> [ hex(p) for p in w.FSA_GetInfoByQuery(w.get_fsa_handle(), '/vol/storage_mlc01/usr/packages', 5)[1]]
['0x0', '0x8c000000', '0x600', '0x100000f3', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0']

>>> [[ hex(p) for p in w.FSA_GetInfoByQuery(w.get_fsa_handle(), path, 5)[1]] for path in ["/vol/storage_mlc01/usr", "/vol/storage_mlc01/usr/boss", "/vol/storage_mlc01/usr/save", "/vol/storage_mlc01/usr/save/00050010",  "/vol/storage_mlc01/usr/packages", "/vol/storage_mlc01/usr/tmp"]]
[['0x0', '0x8c000000', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0'], ['0x0', '0x8c000000', '0x600', '0x100000f6', '0x400', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0'], ['0x0', '0x8c000000', '0x600', '0x100000f6', '0x400', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0'], ['0x0', '0x8c000000', '0x600', '0x100000f6', '0x400', '0x0', '0x0', '0x0', '0x0', '0x0', '0x39672', '0xa661d700', '0x39672', '0xa661d700', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0'], ['0x0', '0x8c000000', '0x600', '0x100000f3', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0'], ['0x0', '0x8c000000', '0x666', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0']]
```

**Comment by jan-hofmeier** [comment]
fixed in #8612efd0f25d03b7de44b8c9302ecb11826fa45f
